project/SparkBuild.scala

```diff
@@ -366,7 +366,7 @@ object Unidoc {
       .map(_.filterNot(_.getCanonicalPath.contains("executor")))
       .map(_.filterNot(_.getCanonicalPath.contains("python")))
       .map(_.filterNot(_.getCanonicalPath.contains("collection")))
-      .map(_.filterNot(_.getCanonicalPath.contains("sql/catalyst")))
+      .map(_.filterNot(_.getCanonicalPath.contains("apache/spark/sql/catalyst")))
       .map(_.filterNot(_.getCanonicalPath.contains("sql/execution")))
```

Comment:
in light of this, i think we should tighten all the paths. can you put the whole path path for all of the packages listed here?
e.g. org/apache/spark/sql/catalyst, org/apache/spark/executor, ..

Make sure you build it to verify it works as expected.

Thanks.